java/src/jmri/web/servlet/panel/AbstractPanelServlet.java

```diff
@@ -238,21 +238,35 @@ protected void parsePortableURIs(Element element) {
      * @param name user/system name of the signalMast using the icons
      * @return an icons element containing icon URLs for SignalMast states
      */
+    @Deprecated
```

Comment:
Please add a deprecated annotation in the Javadoc as well. Something like: `@deprecated since 4.15.4; use {@link #getSignalMastIconsElement(String, String)} instead` That way we know when it was deprecated and explicitly know what the replacement is.